flavour/azure/Dockerfile.flavour

```diff
@@ -1,17 +1,20 @@
-FROM mcr.microsoft.com/azure-cli:cbl-mariner2.0
+FROM mcr.microsoft.com/azure-cli:azurelinux3.0
 
-RUN yum install -y sudo curl tar unzip && \
-    echo "cloudcontrol ALL=(root)NOPASSWD:/usr/bin/yum *" > /etc/sudoers.d/cloudcontrol && \
-    echo "cloudcontrol ALL=(root)NOPASSWD:/usr/bin/az *" >> /etc/sudoers.d/cloudcontrol && \
-    echo "cloudcontrol ALL=(root)NOPASSWD:/usr/bin/chmod *" >> /etc/sudoers.d/cloudcontrol && \
-    echo "cloudcontrol ALL=(root)NOPASSWD:/usr/bin/cp *" >> /etc/sudoers.d/cloudcontrol && \
-    echo "cloudcontrol ALL=(root)NOPASSWD:/usr/bin/update-ca-trust" >> /etc/sudoers.d/cloudcontrol && \
-    adduser -mr cloudcontrol && \
-    mkdir /home/cloudcontrol/bin && \
+
+RUN <<RUN_SCRIPT
+    ln -s /usr/bin/tdnf yum
```

Comment:
Why is this link required?

Reply:
Because Cloudcontrol uses yum and not tdnf and the link is not available on azurelinux 3.0.

Reply:
Ahh ok, you're right. Must have been an intermediate step. This is obviously superfluous and I'll remove it. It was created in the root dir, btw.